examples/example_notebook.py

```diff
@@ -0,0 +1,22 @@
+import tkinter as tk
+import tkinter.ttk as ttk
+from ttkwidgets import Notebook
+
+
+class MainWindow(ttk.Frame):
+    def __init__(self, master=None):
+        super().__init__(master)
+        self.nb = Notebook(self)
+        self.frames = [tk.Frame(self) for i in range(10)]
+        for i, w in enumerate(self.frames):
+            tk.Canvas(w, width=300, height=300).grid(sticky="nswe")
+            self.nb.add(w, text="Frame " + str(i))
+            w.grid()
+        self.nb.grid()
+
+
+root = tk.Tk()
+root.title("Notebook Example")
+gui = MainWindow(root)
```

Comment:
The idea was to have an example that looks like how a real GUI Tk app should be coded. Most people use a class dedicated to placing the widgets, instead of just using functions, and global variables.

Therefore I think the example I wrote is simple enough in that context.

Reply:
Ok, this example is not complicated but, in my opinion, the goal is not to show how to create a class for the application, but I don't really mind if you want to keep the class.

However, I would like you to use the `closecommand` option in the example because this would make this option clearer, especially about the argument passed to the callback.

Reply:
I've updated the example to fit your requested changes